packages/react-router-dom/modules/NavLink.js

```diff
@@ -27,7 +27,7 @@ const NavLink = ({
   return (
     <Route
       path={escapedPath}
-      exact={exact}
+      parent={!exact}
```

Comment:
Currently, `<NavLink>` does non-exact matching and you have to use `<NavLink exact>` for exact matches. I didn't modify that for this PR, but maybe it should be switched for consistency?